appinfo/info.xml

```diff
@@ -5,6 +5,6 @@
     <description>A simple maps app</description>
     <licence>agpl</licence>
     <author>Sander Brand</author>
-    <version>0.0.6</version>
+    <version>0.0.6.1</version>
```

Comment:
I think the best would be to revert it to 0.0.6 and increase it as soon as we release 0.1